index.bs

```diff
@@ -101,6 +101,14 @@ the severity of layout instability at a particular time.  The method of computin
 it considers both the area of the region impacted by instability and the distance
 by which elements on the page are shifted.
 
+The values exposed by this specification are not meant to be used to serve as
+"layout change observers" for a couple of reasons. First, they are tied to the
+PerformanceObserver, hence the dispatch of the callbacks may be done lazily by
+the user agent if it deems this necessary to avoid impacting performance of the
+site. Second, very small layout shifts may be ignored by the user agent. Thus,
+it is not recommended to run JavaScript relying on this API to make user-visible
+behavioral changes on the layout of the website.
```

Comment:
Minor nitpick on the last sentence here: right now, it sounds like this is just recommending against using the API to make "user-visible-changes on [to?] the layout".  This implicitly suggests that other user-visible behavioral changes (e.g. to login state, or other state) might be fine to condition off of this API.

I think perhaps you really meant to recommend against using the API to make any user-visible behavioral changes **at all*, in response to layout-shift reports from this API.

One possible rephrasing:
"Thus, it is not recommended to rely on this API as a way of running any JavaScript that impacts the user-visible behavior of a website."

Reply:
Sure, that works for me.